src/index.ts

```diff
@@ -11,9 +11,9 @@ const server = new GraphQLServer({
       fragmentReplacements,
       endpoint: process.env.PRISMA_ENDPOINT,
       secret: process.env.PRISMA_SECRET,
-      debug: true,
-    }),
-  }),
-})
+      debug: true
+    })
+  })
+});
 
-server.start({ port: 5000 }, ({ port }) => console.log(`Server is running on http://localhost:${port}`))
+server.start(({ port }) => console.log(`Server is running on http://localhost:${port}`));
```

Comment:
port is not defined

Reply:
Weird, this is my output:

```
~/Projects/prisma-examples/graphcool-server-example [updates|✚ 1⚑ 1]
12:37 $ yarn dev
yarn run v1.3.2
$ npm-run-all --parallel start playground
$ nodemon -e ts,graphql -x ts-node -r dotenv/config src/index.ts
$ graphql playground
[nodemon] 1.14.11
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `ts-node -r dotenv/config src/index.ts`
Serving playground at http://localhost:3000/playground
Server is running on http://localhost:4000
```

Reply:
I do see I need to update the `README` once more for the new playground handling :)